packages/cli/snapshots/integration/generators/openapi-petstore.integration.snapshots.js

```diff
@@ -7,42 +7,52 @@
 
 'use strict';
 
-exports[`openapi-generator specific files generates all the proper files 1`] = `
-export * from './pet.model';
-export * from './new-pet.model';
-export * from './error.model';
-
-`;
-
-
-exports[`openapi-generator specific files generates all the proper files 2`] = `
+exports[`openapi-generator petstore generates all the proper files 1`] = `
 export * from './open-api.controller';
 
 `;
 
 
-exports[`openapi-generator specific files generates all the proper files 3`] = `
+exports[`openapi-generator petstore generates all the proper files 2`] = `
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {operation, param, requestBody} from '@loopback/rest';
 import {Pet} from '../models/pet.model';
 import {NewPet} from '../models/new-pet.model';
 
 /**
  * The controller class is generated from OpenAPI spec with operations tagged
- * by 
- * 
+ * by <no-tag>.
+ *
  */
 export class OpenApiController {
   constructor() {}
 
   /**
    * Returns all pets from the system that the user has access to
-Nam sed condimentum est. Maecenas tempor sagittis sapien, nec rhoncus sem sagittis sit amet. Aenean at gravida augue, ac iaculis sem. Curabitur odio lorem, ornare eget elementum nec, cursus id lectus. Duis mi turpis, pulvinar ac eros ac, tincidunt varius justo. In hac habitasse platea dictumst. Integer at adipiscing ante, a sagittis ligula. Aenean pharetra tempor ante molestie imperdiet. Vivamus id aliquam diam. Cras quis velit non tortor eleifend sagittis. Praesent at enim pharetra urna volutpat venenatis eget eget mauris. In eleifend fermentum facilisis. Praesent enim enim, gravida ac sodales sed, placerat id erat. Suspendisse lacus dolor, consectetur non augue vel, vehicula interdum libero. Morbi euismod sagittis libero sed lacinia.
-
-Sed tempus felis lobortis leo pulvinar rutrum. Nam mattis velit nisl, eu condimentum ligula luctus nec. Phasellus semper velit eget aliquet faucibus. In a mattis elit. Phasellus vel urna viverra, condimentum lorem id, rhoncus nibh. Ut pellentesque posuere elementum. Sed a varius odio. Morbi rhoncus ligula libero, vel eleifend nunc tristique vitae. Fusce et sem dui. Aenean nec scelerisque tortor. Fusce malesuada accumsan magna vel tempus. Quisque mollis felis eu dolor tristique, sit amet auctor felis gravida. Sed libero lorem, molestie sed nisl in, accumsan tempor nisi. Fusce sollicitudin massa ut lacinia mattis. Sed vel eleifend lorem. Pellentesque vitae felis pretium, pulvinar elit eu, euismod sapien.
-
-   * 
-
+Nam sed condimentum est. Maecenas tempor sagittis sapien, nec rhoncus sem
```

Comment:
why do we have a big latin paragraph?

Reply:
That's from one of the openapi specs we use for testing.